src/components/modals/authentification/ModalAuthInitView.tsx

```diff
@@ -0,0 +1,123 @@
+
+/**
+ * Modal vue de base avec boutons google, facebook, apple non actifs
+ */
+type Props = {
+  setContenuModal: React.Dispatch<React.SetStateAction<"init" | "connexion" | "inscription" | 'password-forget'>>
+};
+
+const ModalAuthInitView = ({ setContenuModal}: Props ) => {
+
+  return (
+    <>
+      <div className="flex flex-col items-center w-full px-4 pb-6">
+        <h1 className="text-2xl text-center">Bienvenue !</h1>
+
+        <button
+          type="button"
+          role="button"
```

Comment:
Amélioration repérée par Sonarlint : The element button has an implicit role of button. Defining this explicitly is redundant and should be avoided.sonarlint(typescript:S6822)

Reply:
corrigé